test/integration/metricspullthrough/metricspullthrough_test.go

```diff
@@ -138,7 +138,7 @@ func TestPullthroughBlob(t *testing.T) {
 			values: []string{`type="Miss"`},
 		},
 		{
-			name:   "imageregistry_pullthrough_repository_duration_seconds",
+			name:   "imageregistry_pullthrough_repository_duration_seconds_count",
```

Comment:
@dmage did we rename these metrics, or is this an artifact of the prometheus bump?

Reply:
an artifact of the prometheus bump, now we need to define Objectives for summaries. I've copied old defaults to avoid behaviour changes.